src/linter/block.go

```diff
@@ -582,7 +582,9 @@ func (b *blockWalker) withNewContext(action func()) *blockContext {
 }
 
 func (b *blockWalker) handleTry(s *ir.TryStmt) bool {
-	contexts := make([]*blockContext, 0, len(s.Catches)+1)
+	var contexts = make([]*blockContext, 0, len(s.Catches)+1)
```

Comment:
Why change `:=` to var?

Reply:
Just so that all variable definitions are the same :)
```go
var contexts = make([]*blockContext, 0, len(s.Catches)+1)
var linksCount int
var finallyCtx *blockContext
```

Reply:
`var` is generally used for "zero init".
Please don't use var with assignments anywhere except global scope.